webpack/assets/javascripts/react_app/components/BreadcrumbBar/components/Breadcrumb.js

```diff
@@ -3,8 +3,16 @@ import PropTypes from 'prop-types';
 import { Breadcrumb as PfBreadcrumb } from 'patternfly-react';
 import 'patternfly-react/dist/sass/_breadcrumb.scss';
 
+function truncate(text, length) {
```

Comment:
IMHO this function should be in a helper, because we would reuse it in other places.

Reply:
Moved to helpers